EVG-7354: schedule patch PR corrections

tgrander opened the pull request.

john-m-liu commented on graphql/resolvers.go:
```diff
@@ -638,10 +638,12 @@ func (r *mutationResolver) SchedulePatch(ctx context.Context, patchID string, re
+	if err != nil {
+		// FindVersionById does not distinguish between nil version err and db err; therefore must check that err
+		// does not contain nil version err values before sending InternalServerError
+		if !strings.Contains(err.Error(), strconv.Itoa(http.StatusNotFound)) {
```
You should also handle the error being not found here, currently you'll pass a null version into the function below if this happens

tgrander replied:
the flow you describe is intentional. version can be nil.

tgrander commented:
> There are some unaddressed comments from before. For the one that you couldn't get to work, the following diff should work (rename the extension to .diff to git apply it). The reasoning for that is you're setting global state twice by calling functions that the constructor for the testutil package also calls
> 
> [diff.txt](https://github.com/evergreen-ci/evergreen/files/4355061/diff.txt)

THANK YOU!

john-m-liu commented on graphql/integration_atomic_test.go:
```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/evergreen-ci/evergreen/model/user"
 	"github.com/evergreen-ci/evergreen/service"
 	"github.com/evergreen-ci/evergreen/testutil"
+	_ "github.com/evergreen-ci/evergreen/testutil"
```
This is already imported above, so you don't need this line

tgrander commented:
> Re: the change in graphql/tests/taskLogs/results.json, I'm not sure what you mean but it's added by testutil. That's the name of a package - are you talking about a specific utility?

using `ConfigureIntegrationTest` changed the result of the test on its own because it prepended a JIRA link. i mentioned it to @SupaJoon (the author of the original test) and he said it seemed safe to update the result.

SupaJoon commented:
> > Re: the change in graphql/tests/taskLogs/results.json, I'm not sure what you mean but it's added by testutil. That's the name of a package - are you talking about a specific utility?
> 
> using `ConfigureIntegrationTest` changed the result of the test on its own because it prepended a JIRA link. i mentioned it to @SupaJoon (the author of the original test) and he said it seemed safe to update the result.

Updating the Environment changed the output of this function call: https://github.com/evergreen-ci/evergreen/blob/master/rest/model/event.go#L34 
which we use to generate the Jira link
